packages/core/core/src/Parcel.js

```diff
@@ -104,6 +119,20 @@ export default class Parcel {
     );
 
     this.runPackage = this.farm.mkhandle('runPackage');
+
+    if (this.options.serve) {
+      // Not sure if the server should even be mentioned in the core?
+      // Perhaps it should be part of the cli?
+      this.server = await serve(this, this.options.serve);
+    }
+
+    if (this.options.hot) {
```

Comment:
Not sure if this should be part of the core?

Reply:
Same as server, perhaps this could be some kind of reporter or use some events the reporter PR implements instead of the ones I added.

Tried to keep this PR to as little edits to core as possible